sdk/containerregistry/test-resources.json

```diff
@@ -0,0 +1,71 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+        },
+        "tenantId": {
+            "type": "string",
+            "metadata": {
+                "description": "The tenant ID to which the application and resources belong."
+            }
+        },
+        "testApplicationOid": {
+            "type": "string",
+            "metadata": {
+                "description": "The client OID to grant access to test resources."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location of the resource. By default, this is the same as the resource group."
+            }
+        }
+    },
+    "variables": {
+        "baseName": "[parameters('baseName')]",
+        "apiVersion": "2019-05-01",
+        "endpointValue": "[format('https://{0}.azurecr.io', parameters('baseName'))]"
```

Comment:
Any concern (and if so, probably don't need to solve it now) about other clouds and different domain suffixes here? If so, you should try to avoid formatting the URI and instead, if the control-plane response data contains it, pull the endpoint out of it, like I did [here](https://github.com/Azure/azure-sdk-for-net/blob/04df5c5df3913015155392ef7b86f4e2133442aa/sdk/keyvault/test-resources.json#L252).

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/19486 to track.